flask_swagger.py

```diff
@@ -88,7 +120,8 @@ def _extract_array_defs(source):
     return defs
 
 
-def swagger(app, prefix=None, process_doc=_sanitize, template=None):
+def swagger(app, prefix=None, process_doc=_sanitize,
+            from_file_keyword='swagger_from_file', template=None):
```

Comment:
I would prefer from_file_keyword to be default None in which case skip the find_from_file process